### PR TITLE
Fix for Pod Log Fullscreen, toolbar has black background color

### PR DESCRIPTION
**Pull request by dtaylor113**
Fix for https://bugzilla.redhat.com/show_bug.cgi?id=1642896
**Before:**
image
**After:**
image

### CODE REVIEW

File: frontend/public/components/utils/_resource-log.scss [hunk 1]
```diff
@@ -7,5 +7,6 @@
   .co-toolbar {
     padding-left: 10px;
     padding-right: 10px;
+    background-color: $color-pf-white;
```
Comment (rhamilto) [inline frontend/public/components/utils/_resource-log.scss]: Nit: alpha.

Comment (dtaylor113) [reply frontend/public/components/utils/_resource-log.scss]: fixed

**Comment by dtaylor113** [comment]
jenkins build

**Comment by dtaylor113** [comment]
jenkins rebuild

**Pull request merged**